src/codegen/function_body_chunk.rs

```diff
@@ -367,6 +346,18 @@ impl Builder {
         }
     }
 
+    fn is_uninitialized_var(
+        &self,
+        name: &String,
+        uninitialized_vars: &mut Vec<(String, bool)>,
```

Comment:
```
 error: writing `&mut Vec` instead of `&mut [_]` involves a new object where a slice will do
Error:    --> src/codegen/function_body_chunk.rs:352:29
    |
352 |         uninitialized_vars: &mut Vec<(String, bool)>,
    |                             ^^^^^^^^^^^^^^^^^^^^^^^^ help: change this to: `&mut [(String, bool)]`
```